src/definitions/behaviors/form.js

```diff
@@ -1165,6 +1165,17 @@ $.fn.form = function(parameters) {
                 }
               }
             });
+          },
+          optional: function(identifier, bool) {
+            bool = (bool !== undefined)
```

Comment:
Is it intended to use `null` ?

Reply:
@exoego 
I thought it makes more sense to set it as true by default in case you omit that argument: `$('.ui.form').form('set optional', fieldName)`
Should I change this?

Reply:
i second @Giandrop. "set optional" already sounds to enable it by default ( -> `true`). So i think he should not change anything here. @exoego Or did i miss your point here?

Reply:
`null` is false-y value, so it is weird for me `null` is handled as `true`.

Reply:
I wonder what are the use cases users want to set string, int or null to `field.optional`.

Reply:
Ah, i see. So should he change it to
`bool = (bool !== false);` then?

Reply:
I think it's better to check whether the argument is boolean if it expects only boolean.

Reply:
@ko2in So, you agree to my suggestion? Or do you have another suggestion what/how to change?
@exoego  Your final decision about any required change?

Reply:
@lubber-de I agree your suggestion.

Reply:
`!== false` looks good to me

Reply:
@Giandrop Could you please change this line accordingly ? 😉